root/etc/e-smith/templates/etc/shorewall/rules/40services

```diff
@@ -1,8 +1,45 @@
 {
+    # support for old access property
+    sub accessToZones
```

Comment:
Why do we need ``accessToZones()`` in templates if we have the migration fragment? It should not be needed: the ``access`` prop does not contain ``public`` ``private`` or ``none`` any more.